base/db/R/convert.input.R

```diff
@@ -736,7 +737,7 @@ convert.input <-
     log_format_df = function(df){
       df  %>%
         format() %>%
-        rbind(colnames(.), .) %>%
+        df %>% format() %>% rbind(colnames(.data), .data) %>%...
```

Comment:
Pretty sure this is not valid R code to pipe something into `...`.  Even if this line is valid, then the next line will error because nothing is getting piped into `purrr::reduce()`.

Reply:
`rbind(colnames(df %>% format()), df %>% format())` @Aariq will this code work,

Reply:
`rbind(colnames(df %>% format()), df %>% format())` @Aariq will this code work? I tried not to use pipe

Reply:
yeah, I think that works.  You could also do 
```suggestion
        rbind(colnames(format(df)), format(df))
```
I'm still a little confused about why this function exists.  Why would you want the column names to print twice?

Reply:
``` r
df <- data.frame(x = 1:3, y = c("a", "b", "c"))

#why would you want this?
rbind(colnames(format(df)), format(df))
#>    x y
#> 1  x y
#> 11 1 a
#> 2  2 b
#> 3  3 c

#this does the same thing, but also WHY?
rbind(colnames(df), df)
#>   x y
#> 1 x y
#> 2 1 a
#> 3 2 b
#> 4 3 c
```

<sup>Created on 2022-08-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
Feels like someone tried to re-invent the wheel here.  `print(df, row.names = FALSE)` does something similar. 🤷‍♂️

Reply:
@Aariq should I replace `df %>% format() %>% rbind(colnames(.data), .data) %>%...`  with `rbind(colnames(df), df) %>%` as it does the same thing? or should commit the suggestion?

Reply:
I don't think it does anything if you don't supply it with any arguments other than maybe strip attributes, but I guess I'd just leave `format()` in since there aren't any tests for this function.